catanatron/game.py

```diff
@@ -274,6 +279,13 @@ def execute(self, action, initial_build_phase=False):
             self.resource_deck += ResourceDeck.development_card_cost()
 
             action = Action(action.player, action.action_type, development_card)
+        elif action.action_type == ActionType.PLAY_YEAR_OF_PLENTY:
+            cards_selected = action.value #Assuming action.value is a resource deck
```

Comment:
We can return this as a resource_deck.to_array() if you want as well to make it nicer for the logs